.buildkite/hooks/pre-command

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/env-scripts/env.sh
+source .buildkite/env-scripts/util.sh
+source .buildkite/env-scripts/win-env.sh
+
+
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "filebeat" ]]; then
+  if [[ ${PLATFORM_TYPE} = MINGW* ]]; then
+    install_python_win
```

Comment:
For Windows, you can also use the `pre-command.bat`, see https://buildkite.com/docs/agent/v3/hooks#job-hooks-on-windows

https://github.com/elastic/elasticsearch/blob/main/.buildkite/hooks/pre-command.bat is how the Elasticsearch team uses it.

Reply:
We will create custom images for beats, so this will be removed from the script eventually. I think we can keep it like this for time being.